spyderlib/spyder.py

```diff
@@ -2646,6 +2646,11 @@ def start_open_files_server(self):
                 # To avoid a traceback after closing on Windows
                 if e.args[0] == eintr:
                     continue
+                # handle a connection abort on close error
+                enotsock = (errno.WSAENOTSOCK if os.name == 'nt'
+                            else errno.ENOTSOCK)
+                if e.args[0] == enotsock:

```

Comment:
At the end this is essentially what I added before, but with `return` instead of `continue` ;-)